libs/waterfall/src/lib/components/graph/layout.ts

```diff
@@ -776,17 +874,17 @@ export function fromGraph(graph: readonly Node[]) {
         node.members.forEach(member => {
           if (member.type === 'vertical') {
             rights.push(createVertical(member, undefined, node.id));
-            member.members.forEach(subMember => {
-              rights.push(createRight(subMember, undefined, member.id));
+            member.members.forEach((subMember, index) => {
+              rights.push(createRight(subMember, undefined, member.id, index));
```

Comment:
for better consistency.
```suggestion
            member.members.forEach((subMember, order) => {
              rights.push(createRight(subMember, undefined, member.id, order));
```